mongo/datadog_checks/mongo/metrics.py

```diff
@@ -309,8 +309,7 @@
     'collection.max': GAUGE,
     'collection.maxSize': GAUGE,
     'collection.storageSize': GAUGE,
-    'collection.nindexes': GAUGE,
-    'collection.indexSizes': GAUGE,
```

Comment:
`collection.indexSize` is collected on its own, it does not have to be included here and leads to errors with the PR as the submission logic is now generic instead of being duplicated three times.